src/game_frontend/src/components/ui/Container/index.css

```diff
@@ -0,0 +1,29 @@
+.thank-you-wrapper {
+    background-color: blue; /* Added here opacity to background blue */
+    box-shadow:
+        /* Inner blue layer with 50% opacity */ 30px 0 blue,
+      /*  right shadow */ -30px 0 blue,
+      /*  left shadow */ 0 -30px blue,
+      /*  top shadow */ 0 30px blue,
+      /*  bottom shadow rgba(0, 0, 255, 0.5) */
+        /* Middle white layer */ 60px 0 white,
+      -60px 0 white, 0 -60px white, 0 60px white,
+      /* White spread */ 0 0 0 30px white,
+      /* Outer red-orange layer  0px 0 #ff4500, -0px 0 #ff4500, 0 0px #ff4500, */
+      0 -0px #ff4500;
+    margin: 18px auto;
+    text-align: center;
+    display: inline-block;
+    padding: 0.4em 1.3em;
+    font: 1.4em "Press start 2P";
```

Comment:
Use Proper Font as in Figma

Reply:
sir ye maine apne sey add ni kia hai ye already tha code me wo vertex ke liye